docs/api/ref/DesktopAgent.md

```diff
@@ -376,21 +376,23 @@ await fdc3.open('myApp', context);
 ```ts
 raiseIntent(intent: string, context: Context, app?: TargetApp): Promise<IntentResolution>;
 ```
-Raises a specific intent against a target app.
+Raises a specific intent for resolution against apps registered with the desktop agent. 
 
-The desktop agent will resolve the correct app to target based on the provided intent name and context data.
+The desktop agent will resolve the correct app to target based on the provided intent name and context data. If multiple matching apps are found, the user should be presented with a resolver UI allowing them to pick an app.
```

Comment:
I am not happy with this statement being in the docs/spec as a should. I think a resolver UI is just one possible way to deal with intent resolution.

Some other options I can think of:
* Pick the most recently used option.
* User preferences that control default apps that are configured somewhere.